File: cardculatorccmodule/CardculatorCCModule.m
#import "CardculatorCCModule.h"

@implementation CardculatorCCModule

//Return the icon of your module here
- (UIImage *)iconGlyph
{
	return [UIImage imageNamed:@"CCIcon" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
}

//Return the color selection color of your module here
- (UIColor *)selectedColor
{
	return [UIColor blueColor];
}

- (BOOL)isSelected
{
  return NO;
}

- (void)setSelected:(BOOL)selected
{
  [super refreshState];
  [[NSNotificationCenter defaultCenter] 
        postNotificationName:@"PresentCalculator" 
        object: nil];
}

@end

File: cardculatorccmodule/Sources/CardculatorCCModule/CardculatorCCModule.swift
import Orion
import CardculatorCCModuleC

class CardculatorCCModule: CCUIToggleModule {
    
    @objc override var iconGlyph: UIImage? {
        UIImage(named: "CCIcon", in: Bundle(for: type(of: self)), compatibleWith: nil)
	// return [UIImage imageNamed:@"CCIcon" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
    }
    @objc override var selectedColor: UIColor? {
        .blue
    }
    @objc override var isSelected: Bool {
        get {
            false
        }
        set {
            super.refreshState()

            remLog("selected")
            NotificationCenter.default.post(name: .init("CCPresentCalculator"), object: nil)
        }
    }
}